homeassistant/components/dominos/__init__.py

```diff
@@ -71,7 +71,16 @@ def setup(hass, config):
     entities = []
     conf = config[DOMAIN]
 
-    hass.services.register(DOMAIN, "order", dominos.handle_order)
+    hass.services.register(
+        DOMAIN,
+        "order",
+        dominos.handle_order,
+        vol.Schema(
+            {
+                vol.Required(ATTR_ORDER_ENTITY): cv.entity_ids,
```

Comment:
Or maybe this should be `cv.comp_entity_ids` ?

Reply:
I don't think we have to match `none` or `all` here... If one uses `all` and have 2 entities, it would create two orders... `none` is uh... no pizza :)

Reply:
So `cv.entity_ids` here is good then, right?

Reply:
yes